File: team.py
n=int(input())
lst=[]
for i in range(0,n) :
    for j in range(0,3) :
        a=int(input())
        lst.append(a)
print(lst);
number=0
count=1
increase=0
for i in range(0,n):
    if lst[i]==1 :
        number=number+1
    count=count+1
    if count == 3 :
        if number>=2 :
            increase=1+increase
        count=0
        number=0
    print(count,increase,number)
print (increase)

File: team.c
#include <stdio.h>
 
int main(){
    int n,i,count=0,increase=0,temp=0;
    scanf("%d",&n);
    int a[n*3];
    for(i=0;i<n*3;i++)
    {
        scanf("%d",&a[i]);
    }
    
    for(i=0;i<n*3;i++)
    {   
        temp=temp+a[i];
        count=1+count;
        if(count==3)
            {
            count=0;
            if(temp>=2)
                increase=1+increase;
            temp=0;
            }
    }
    printf("%d",increase);
    return 0;
}

